samples/csharp/MicrosoftGraphSDKBeta/User.cs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License.
 
 using Microsoft.Graph.Beta;
+using Microsoft.Graph.Beta.Models;
+using MicrosoftGraphSDK;
```

Comment:
I don't think it is needed, can you please remove and test the sample to make sure it works?